packages/node/src/policy-gen/generate.js

```diff
@@ -94,17 +88,17 @@ const generate = async (
     log = defaultLog,
     dev = false,
     trustRoot = DEFAULT_TRUST_ROOT_COMPARTMENT,
+    conditions = new Set(),
```

Comment:
I don't see this being used anywhere and failed to come up with a good idea why we'd need to use it.

Reply:
it's just passed through to `loadCompartmentMap` below, but re: the other comment about `conditions`, I intend to remove it.